units/src/locktime/absolute.rs

```diff
@@ -384,6 +386,36 @@ impl From<ConversionError> for ParseError {
     fn from(value: ConversionError) -> Self { Self::Conversion(value.input.into()) }
 }
 
+#[cfg(feature = "arbitrary")]
+impl<'a> Arbitrary<'a> for Height {
+    fn arbitrary(u: &mut Unstructured<'a>) -> arbitrary::Result<Self> {
+        let choice = u.int_in_range(0..=2)?;
+        match choice {
+            0 => Ok(Height::MIN),
+            1 => Ok(Height::MAX),
+            _ => {
+                let h = u.int_in_range(0..=Height::MAX.to_consensus_u32())?;
+                Ok(Height::from_consensus(h).unwrap())
+            }
+        }
+    }
+}
+
+#[cfg(feature = "arbitrary")]
+impl<'a> Arbitrary<'a> for Time {
+    fn arbitrary(u: &mut Unstructured<'a>) -> arbitrary::Result<Self> {
+        let choice = u.int_in_range(0..=2)?;
+        match choice {
+            0 => Ok(Time::MIN),
+            1 => Ok(Time::MAX),
+            _ => {
+                let t = u.int_in_range(LOCK_TIME_THRESHOLD..=Time::MAX.to_consensus_u32())?;
```

Comment:
This doesn't seem right.  MAX is defied as `LOCK_TIME_THRESHOLD - 1`. so MAX is less than `LOCK_TIME_THRESHOLD` if I'm reading this right.  So you have a range of only two values and the larger value in the range first?

Reply:
I think you are looking at `Height::MAX`; `Time::MAX` is defined as `Time(u32::MAX)`

Reply:
Oh ok, then that checks out.  Sorry for the confusion.